### PR TITLE
Add DisasterRecoveryMode option to ClusterPair requests

### PR DESCRIPTION
**Pull request by disrani-px**
Signed-off-by: Dinesh Israni <disrani@portworx.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: api/api.proto [hunk 1]
```diff
@@ -3414,6 +3414,8 @@ message ClusterPairCreateRequest {
   string remote_cluster_token = 3;
   // Set the new pair as the default
   bool set_default = 4;
+  // If the pair is being created for DR mode
+  bool disaster_recovery_mode = 5;
```
Comment (talakad) [inline api/api.proto]: Suggestion:Should this be a enum rather than bool,  to allow any flavors of disaster recovery?

Comment (disrani-px) [reply api/api.proto]: Yeah, that's a good point. What should the types be though? Should the name be Mode, and different modes be Default and DisasterRecovery for now?

Comment (talakad) [reply api/api.proto]: Yeah that sounds great. default would be disaster_recovery_mode_off.

Comment (disrani-px) [reply api/api.proto]: Updated. Can you take a look again?

Comment (talakad) [reply api/api.proto]: gook great!.

**Pull request merged**